packages/react/src/account/useAccountManagers.ts

```diff
@@ -0,0 +1,42 @@
+import type { AccountManager, AccountManagersRequest, Paginated } from '@lens-protocol/graphql';
+import { AccountManagersQuery } from '@lens-protocol/graphql';
+
+import type { ReadResult, Suspendable, SuspendableResult, SuspenseResult } from '../helpers';
+import { useSuspendableQuery } from '../helpers';
+
+export type AccountManagersArgs = AccountManagersRequest;
+
+/**
+ * Fetch Account Managers.
+ *
+ * This signature supports React Suspense:
+ *
+ * ```tsx
+ * const { data } = useAccountManagers({ suspense: true });
+ * ```
+ */
+export function useAccountManagers(
+  args: AccountManagersArgs & Suspendable,
+): SuspenseResult<Paginated<AccountManager>>;
+
+/**
+ * Fetch Account Managers.
+ *
+ * ```tsx
+ * const { data, loading } = useAccountManagers();
+ * ```
+ */
+export function useAccountManagers(
+  args: AccountManagersArgs,
```

Comment:
I didn't check this in detail, but wherever the request is optional, and from the example `useAccountManagers()` it seems so, this param should optional.

Otherwise this will only work with an awkward empty object `useAccountManagers({})` if no specific args are needed.